jcl/source/common/JclSysInfo.pas

```diff
@@ -2606,8 +2606,10 @@ function GetBIOSExtendedInfo: string;
 { TODO : the date string can be e.g. 00/00/00 }
 function GetBIOSDate: TDateTime;
 const
-  WinNT_REG_PATH = 'HARDWARE\DESCRIPTION\System';
-  WinNT_REG_KEY  = 'SystemBiosDate';
+  WinNT_REG_PATH_NEW = 'HARDWARE\DESCRIPTION\System\BIOS';
+  WinNT_REG_PATH_OLD = 'HARDWARE\DESCRIPTION\System';
+  WinNT_REG_KEY_NEW  = 'BIOSReleaseDate';
+  WinNT_REG_KEY_OLD  = 'SystemBiosDate';
```

Comment:
I dislike the "new"/"old" naming as it doesn't convey any meaning as to when the change happened. And what if it changes again? Will it then be "NEW_NEW" or "NEW_2"?
I'd rather keep `WinNT_REG_PATH` and `WinNT_REG_KEY` introduce `WIN10_REG_PATH` and `WIN10_REG_KEY` both declared together right after the `const` line